### PR TITLE
Feature/router

### PR DESCRIPTION
**Pull request by knees4bees**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [x] Feature (to add router)
- [x] Refactor (to add planet images rather than icons on detail view)
- [x] Style (very minor styling)
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
* Add Router
* Add home-ish button-ish

### What ticket(s) do the changes resolve?
#17

Please Link Issues to this PR as necessary.
https://github.com/ConnorAndersonLarson/PlanetParty/issues/17#issue-855381014

### Notes for the Reviewer
Wrapping the planets in our Planetarium component in `<Link>`s added some annoying style side effects. Same with making the Planet Party! title a `<Link>`.

### How to Test Changes?
Confirm that click on a planet takes you to a new URL, that you can get back home by clicking on the title or on the back-ish button-ish, that actual planet images appear on the planet detail view, and that typing in a bad url (e.g. `/vulcan`) just takes you to the home page (but we can change that behavior if we want!)

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -36,11 +37,37 @@ class App extends React.Component<{}, AllData> {
...
+        <Route exact path="/" render={ () => {
+          return (
+            <main>
+              <SortBox updateSort={this.updateSort} />
+              <Planetarium allPlanets={this.state.allPlanets} />
+            </main>
+          )
+        }} />
+
+        <Route path="/:name" render={ ({ match }) => {
```
Comment (alexthompson207) [inline src/components/App/App.tsx:50]: Should this be exact path?

**Review by alexthompson207 — APPROVED** [review]
This looks great! Really good work with this!

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks good from our group walkthrough!

**Comment by alexthompson207** [comment]
Fixed merge conflicts, everything looks great!

**Pull request merged**